classes/mcPy/McPy.py

```diff
@@ -32,6 +32,35 @@ def _launch(parser: Parser):
         probe.enable()
         logging.info("Blackfire Enabled!")
 
+    print("Loading data files, please wait ...")
```

Comment:
Use the logging module, for console consistency

Reply:
Good find